### PR TITLE
Placeholders for disaggregation

### PR DESCRIPTION
**Pull request by bl-young**
Initiating this pull request per the conversation today so that all branches moving forward will have the placeholders for the disaggregation.
-`prepareEEIOModel` renamed to `loadIOData`
-economic disaggregation placeholder in `loadIOData` (BuildModel.R)
-satellite table disaggregation placeholder in LoadSatellites
-new DisaggregateFunctions.R

### CODE REVIEW

**Comment by bl-young** [comment]
There was discussion of moving where loadsattables is called, but I missed if we decided where it should go

File: R/DisaggregateFunctions.R [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+#' @param model A complete EEIO model: a list with USEEIO model components and attributes.
+#' @param sattable A standardized satellite table with resource and emission names from original sources.
+#' 
+#' @return A standardized satellite table with old sectors removed and new sectors added.
+disaggregateSatelliteTable <- function (model, sattable){
+  
+  #Disaggregate satellite tables
+  
+  return (sattable_disaggregated)
+}
```
Comment (WesIngwersen) [inline R/DisaggregateFunctions.R:12]: remove space after return 
Add new line to end of file

**Comment by MoLi7** [comment]
Consider move `loadIOData` to LoadIOTables.R for improved code organization.

**Comment by bl-young** [comment]
@MoLi7 ok made that move

**Pull request merged**